### DISCUSSION TITLE
LANraragi keeps crashing. Did I set it up improperly?

### DISCUSSION THREAD

**Discussion by asdfsheep** [Difegue/LANraragi#421] [Q&A]
Hey everyone! I'm super new to this and I am not really familiar with all those networking terms and shenanigans so please bear with me.

So I followed the documentation and set it up using the Windows Installer Package and got it running. When I first started, it told me to select a Contents Folder and Thumbnail Folder. I set the Contents Folder to a folder in my external HDD, while having the Thumbnail Folder saved in my desktop. I have about 6 thousand CBR files in the Contents Folder. I started it and LANraragi and localhost3000 works perfectly fine and was updating and generating the thumbnails. However, once all files have been uploaded, when I try to read the archives, it doesn't load and is stuck loading forever. When I go to another page of the archives, it stops loading. When I turn off LANraragi and turn it back on, LANraragi crashes and I need to restart my entire laptop in order for it to work again.

What seems to be the problem? Is my folder too big?

Sorry if this seems like a stupid question, but I am really desperate to get it to work and I have no idea how :(

**Comment by Difegue** [comment]
Sounds like the server is choking when it tries to extract one of your CBRs? 
You might want to try checking in the log view of the application for any more precide errors.

IIRC, you shouldn't have to restart the machine if you use the "stop server" button in the windows UI since that hard shutdowns it.

**Reply by asdfsheep** [reply]
Oh its the LANraragi app crashing when I press stop server and then start server again. When I force quit it, it would not open again, which forces me to restart my PC for it to open the app

**Reply by Difegue** [reply]
Oh, that's more troublesome than I thought -- The Karen.exe bootstrapper is quite small so it doesn't have much in terms of error logging. Do you encounter the same issues if you run an older version of the app, like 0.7.6 ?  

I made a few changes to the windows app to accommodate for the new thumbnail directory and mightve broken something in the process. 💦
